Makefile

```diff
@@ -179,15 +178,18 @@ TARGET_ARCH_DEBIAN = $(shell ./scripts/build/architecture-convert.sh -r $(TARGET
 TARGET_ARCH_REDHAT = $(shell ./scripts/build/architecture-convert.sh -r $(TARGET_ARCH) -t redhat)
 TARGET_ARCH_APPIMAGE = $(shell ./scripts/build/architecture-convert.sh -r $(TARGET_ARCH) -t appimage)
 TARGET_ARCH_ELECTRON_BUILDER = $(shell ./scripts/build/architecture-convert.sh -r $(TARGET_ARCH) -t electron-builder)
+TARGET_PLATFORM_PKG = $(shell ./scripts/build/target-convert.sh -r $(TARGET_PLATFORM) -t pkg)
```

Comment:
Ahhh, I've just spotted that Appveyor says:
```
make: ./scripts/build/target-convert.sh: Command not found
mkdir dist/Etcher-cli-1.0.0+b67afbe-win32-x64
./node_modules/.bin/npx pkg --output dist/Etcher-cli-1.0.0+b67afbe-win32-x64/etcher.exe -t node6--x64 dist/Etcher-cli-1.0.0+b67afbe-win32-x64-app/lib/cli/etcher.js
```
and Travis says:
```
make: ./scripts/build/target-convert.sh: Command not found
mkdir dist/Etcher-cli-1.0.0+b67afbe-linux-x64
./node_modules/.bin/npx pkg --output dist/Etcher-cli-1.0.0+b67afbe-linux-x64/etcher -t node6--x64 dist/Etcher-cli-1.0.0+b67afbe-linux-x64-app/lib/cli/etcher.js
```
so it's **not** me being stupid, and this indeed isn't quite working as expected - note the `node6--x64` strings with the missing platform specifier.